### DISCUSSION TITLE
Larger Gem Tokens

### DISCUSSION THREAD

**Discussion by Lord0nyx** [gameserverapp/Platform#879] [Suggestions]
Can we please get larger denomination Token Gems? We have some auctions that reach up to 181k Tokens.. thats a lot of 1k gems if being done player to player..and average costs are from 5-30k tokens.
It would be nice to have 5k, 10k, 25k and 50k options. But even just adding 5k and 10k would be a huge Improvement.

**Comment by mvaessen** [comment]
hi, which game are you referring too? ASA or ASE?

**Reply by Lord0nyx** [reply]
Both would be ideal I imagine, since our token and bush berry prices are roughly the same, but primarily now I was thinking of ASE. I figured it would be available platform wide if you implement something like that.

**Reply by mvaessen** [reply]
There might be a chance that this gets added for ASA. But ASE is EOL.

There are mods available that can trigger webhooks. You can use those together with GSA webhooks to create custom token gems for ASA. Im not aware of any mods that can do that for ASE.

Since this suggestion is primarily for ASE, am I going to reject the suggestion. There are no plans to work on the ASE mod. The ASA mod will soon be open sourced, which lets you customize everything you want. I recommend keeping an eye on GSA discord for updates about this.

**Reply by Lord0nyx** [reply]
I mean .. I guess that's better than nothing. ASE will just have to make do with $1000 bills. I suppose in a few years everyone will switch over anyway. ASE is only surviving on content superiority at the moment.
